Magento_Theme/templates/header/search.phtml

```diff
@@ -87,7 +87,7 @@ $helper = $this->helper('Magento\Search\Helper\Data');
         "quickSearch" :
         {
             "formSelector":"#search_mini_form",
-            "url":"<?php /* @escapeNotVerified */ echo $block->getUrl('search/ajax/suggest'); ?>",
+            "url":"<?= $block->escapeUrl($block->getUrl('search/ajax/suggest')); ?>",
```

Comment:
I think here we should extra escapeJs to wrap escapeUrl. but please later test in on frontend.

Reply:
Added and tested.